docs/windows.md

```diff
@@ -0,0 +1,227 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports Windows worker Node. On Windows Node, Antrea sets up an overlay
+network to forward packets between nodes. Currently VXLAN and STT tunnels are
```

Comment:
We could add NetworkPolicy too like
"Antrea sets up an overlay network between Nodes and implements NetworkPolicies".

Reply:
nodes -> Nodes.

Reply:
Done.